Formula/nexus.rb

```diff
@@ -1,9 +1,9 @@
 class Nexus < Formula
   desc "Repository manager for binary software components"
   homepage "https://www.sonatype.org/"
-  url "https://sonatype-download.global.ssl.fastly.net/repository/downloads-prod-group/oss/nexus-2.14.19-01-bundle.tar.gz"
-  version "2.14.19-01"
-  sha256 "5bf321e40b219756441fbdb9fa0d270493a25640af67076111ccd151ab007b7e"
+  url "https://github.com/sonatype/nexus-public/archive/release-3.28.1-01.tar.gz"
```

Comment:
maybe this would be right artifact location?
`http://download.sonatype.com/nexus/3/latest-mac.tgz`

Reply:
That's unversioned, we can't use that.